third_party/nvfuser/csrc/tensor_view.cpp

```diff
@@ -1513,12 +1501,23 @@ TensorViewBuilder& TensorViewBuilder::dtype(DataType dtype) {
   return *this;
 }
 
-TensorViewBuilder& TensorViewBuilder::contiguity(std::vector<bool> contiguity) {
-  TORCH_CHECK(contiguity_.empty(), "Attempting to reset contiguity");
+TensorViewBuilder& TensorViewBuilder::contiguity(
```

Comment:
Can you assert the contiguity vector has the same length as the number of dimensions (including broadcasts)?

Reply:
added